src/site/_includes/js/test/eligibility-programs.test.js

```diff
@@ -2214,4 +2214,54 @@ describe('Program eligibility', () => {
         .isEligibleIf('income.wages').isAtMost(maxIncome);
     });
   });
+  describe('HUD VASH Program', () => {
```

Comment:
Many of these tests have some leftover copy/paste code (e.g. `noFeeIdResult`, `vaDisabilityResult`)

Reply:
Fixed in [6ea421a](https://github.com/theunitedeffort/theunitedeffort.org/pull/629/commits/6ea421af31bf408192baca9fd00c69829bbdfa8c)